tests/loadtesting/admin.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import os
+from locust import HttpUser, task, between
+
+
+class QuickstartUser(HttpUser):
+    # Our class defines a wait_time function that will make the simulated users
+    #  wait between 5 and 9 seconds after each task is executed
+    wait_time = between(5, 9)
+
+    @task
+    def index_page(self):
+        self.client.get("https://localhost/admin/#/", verify=False)
```

Comment:
No need to hard-code the full URL here, just use the endpoint without `https://localhost' and it should use the host you provided?

Reply:
This will work with the first two enpoints and they don't require a port but in the last endpoint, it didn't work due to the presence of the port number